dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -813,7 +813,7 @@
 			label = "USBD_2";
 		};
 
-		usdhc1: usdhc@402c0000 {
+		sdhc0: usdhc1: usdhc@402c0000 {
```

Comment:
I think tests/samples should rely on aliases, this is changing soc dts files for testing only. I don't think this can scale.